src/indicator.js

```diff
@@ -112,14 +118,17 @@ export default class RunCatIndicator extends PanelMenuButton {
 
 	repaintUi() {
 		/** @type {CharacterState} */
-		const characterState = this.#data?.cpu > this.#settings.idleThreshold ? 'active' : 'idle'
+		const isActive = this.#data?.cpu > this.#settings.idleThreshold || this.#settings.reverseSpeed;
```

Comment:
Please run `npm run test:lint` command. `;` is 100% redundant

Reply:
https://github.com/win0err/gnome-runcat/actions/runs/7503777096/job/20453894666?pr=44

Reply:
> Please run `npm run test:lint` command. `;` is 100% redundant

Sorry, i completely forgot about it. I fixed all lint errors.